src/test/java/auto_ckz/domain/repairhistory/RepairHistoryTest.java

```diff
@@ -0,0 +1,131 @@
+package auto_ckz.domain.repairhistory;
+
+import auto_ckz.config.JpaConfigForTest;
+import com.github.springtestdbunit.DbUnitTestExecutionListener;
+import com.github.springtestdbunit.annotation.DatabaseSetup;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+import org.springframework.test.context.support.DirtiesContextTestExecutionListener;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.List;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = {JpaConfigForTest.class})
+@TestExecutionListeners({ DependencyInjectionTestExecutionListener.class,
+        DirtiesContextTestExecutionListener.class,
+        TransactionalTestExecutionListener.class,
+        DbUnitTestExecutionListener.class })
+@DatabaseSetup("/testData/toDoData.xml")
+public class RepairHistoryTest {
+
+    @Autowired
+    private RepairHistoryRepository repository;
+
+    @Test
+    public void NoEntries_findByCarId_ShouldReturnEmptyList() {
+        List<RepairHistory> repairHistoryEntries = repository.findByCarId(null);
+        assertThat(repairHistoryEntries.size(), is(0));
+    }
+
+    @Test
+    public void findByDate_ShouldReturnOneRepairHistoryEntry() throws ParseException {
+
+        String date_s = "2014-02-11";
+        SimpleDateFormat dt = new SimpleDateFormat("yyyy-MM-dd");
+        java.util.Date date = dt.parse(date_s);
+        java.sql.Date sqlDate =  new java.sql.Date(date.getTime());
```

Comment:
Może zamieniać to od razu na `java.sql.Date`? http://stackoverflow.com/a/18439059